### PR TITLE
feat(evince): complete on XPS and more comic book documents

### PR DESCRIPTION
**Pull request by scop**
https://gitlab.gnome.org/GNOME/evince/-/blob/32e832611b1e60834fc5797311d46b98a6e5f958/help/C/formats.page#L32

### CODE REVIEW

**Review by akinomyoga — APPROVED** [review]

**Comment by akinomyoga** [comment]
A test seems to still have unsupported image formats dropped in #747. I have pushed a commit 1653c6cb to drop unsupported extensions and to add the new extensions .oxps, .xps, .cb7, and .cbt.

File: completions/evince [hunk 1]
```diff
@@ -25,7 +25,7 @@ _evince()
         return
     fi
 
-    _filedir '@(@(?(e)ps|?(E)PS|[pf]df|[PF]DF|dvi|DVI)?(.gz|.GZ|.bz2|.BZ2|.xz|.XZ)|cb[rz]|djv?(u)|tif?(f))'
+    _filedir '@(@(?(e|?(o)x)ps|?(E)PS|[pf]df|[PF]DF|dvi|DVI)?(.gz|.GZ|.bz2|.BZ2|.xz|.XZ)|cb[7rtz]|djv?(u)|tif?(f))'
```
Comment (akinomyoga) [inline completions/evince:28]: I have tried to open a `.oxps.gz` file with `evince` (evince-42.2 on Fedora 36), but it doesn't seem to support `.oxps.gz` while it does support `.ps.gz` and `.pdf.gz`

**Comment by scop** [comment]
Thanks!

File: completions/evince [hunk 2]
```diff
@@ -25,7 +25,7 @@ _evince()
         return
     fi
 
-    _filedir '@(@(?(e)ps|?(E)PS|[pf]df|[PF]DF|dvi|DVI)?(.gz|.GZ|.bz2|.BZ2|.xz|.XZ)|cb[rz]|djv?(u)|tif?(f))'
+    _filedir '@(@(?(e)ps|?(E)PS|[pf]df|[PF]DF|dvi|DVI)?(.gz|.GZ|.bz2|.BZ2|.xz|.XZ)|cb[7rtz]|djv?(u)|tif?(f)||?(o)xps)'
```
Comment (akinomyoga) [inline completions/evince:28]: I think the `|`'s before `?(o)xps` are duplicate.

Comment (scop) [reply completions/evince:28]: Sigh, thanks. Fixed in a79746c62917c25a17b4711e3d8527f2b67b0a5c

**Pull request merged**